### ISSUE TITLE
Kodi automatically starts resuming playback after resuming from sleep

### ISSUE THREAD

**Issue by jddcef** [xbmc/xbmc#18043]
### Describe the bug
I pause a video in Kodi that is playing. It's now Paused.
When I Suspend or Sleep my Windows 10 laptop 
and then waking up or resuming the laptop and unlocking by entering windows password, then Kodi starts playing or resuming automatically by itself.

## Expected Behavior
Kodi shouldn't start playing unless I press play after I have turned on my logged into my laptop again.

## Actual Behavior
Kodi starts playing by itself.
Actually, it usually crashes Kodi, but I haven't caught it in the logs as of yet

## Possible Fix
N/A

### To Reproduce
1. Play video 
2. Pause video.
3. Sleep laptop
4. Wake up laptop
5. Unlock/login.

### Debuglog
The debuglog can be found here:
https://paste.kodi.tv/uranizakil.kodi

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:
I also have a debug log that I took from Windows Feedback Hub, a diagnostics recording of the crash if that helps?

### Your Environment
Used Operating system:

OS Name	Microsoft Windows 10 Home Single Language
Version	10.0.19041 Build 19041

 - Kodi version:
Starting Kodi (18.7 Git:20200520-f14f7a84e3). Platform: Windows NT x86 64-bit
Using Release Kodi x64 build
Kodi compiled 2020-05-20 by MSVC 191627035 for Windows NT x86 64-bit version 10.0 (0x0A000006)
Running on Windows 10, kernel: Windows NT x86 64-bit version 10.0.19041
FFmpeg version/source: 4.0.4-Kodi
Host CPU: Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz, 8 cores available
Desktop Resolution: 1366x768 32Bit at 60Hz
Running with restricted rights
Aero is enabled

Actually, I this also happens on Android version of Kodi.

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by jqdg** [comment]
It looks like this is a duplicate of #15737. I'm also experience this issue and it's very annoying. Is it intended behavior or a bug?

**Comment by imammedo** [comment]
The same happens with KODI 19 (kodi-19.1-1.fc34.x86_64)
Once laptop wakes up, playback resumes immediately and if speakers aren't muted it takes time to unlock and pause the noise machine (which is not acceptable in the office settings). The only way to avoid it so far is to stop playback completely before suspend.
